bitcoin/chainparams.c

```diff
@@ -21,7 +28,10 @@ const struct chainparams networks[] = {
      .max_payment = AMOUNT_MSAT_INIT(0xFFFFFFFFULL),
      /* "Lightning Charge Powers Developers & Blockstream Store" */
      .when_lightning_became_cool = 504500,
-     .testnet = false},
+     .p2pkh_version = 0,
```

Comment:
Should this not be 36 (0x24)?

Reply:
It should be 36, thanks @HashEngineering

Reply:
fixed, as if mergetool was rewritten by the Bitcoin BS lizzards, good catch :-)